akka-distributed-data/src/main/scala/akka/cluster/ddata/ORSet.scala

```diff
@@ -307,6 +333,46 @@ final class ORSet[A] private[akka] (
     }
   }
 
+  private def mergeAddDelta(that: ORSet[A]): ORSet[A] = {
```

Comment:
Need to debug this on paper before I can say anything. Will do this later today.